ui/src/style/chronograf.scss

```diff
@@ -109,7 +109,11 @@
 @import 'src/onboarding/components/SigninForm.scss';
 @import 'src/shared/components/ThresholdsSettings.scss';
 @import 'src/shared/components/ThresholdMarkers.scss';
+<<<<<<< HEAD
 @import 'src/shared/components/DeleteDataForm/DeleteDataForm.scss';
+=======
+@import 'src/shared/components/cloud/CloudOnly.scss';
+>>>>>>> fix(ui/asset-alert): fix asset-alert clipping dashboard contents
```

Comment:
I think this needs to be resolved

Reply:
good catch 🚀 , did save the rebase to accept both 😞

Reply:
gotta love git 😹